src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -94,7 +96,8 @@ export default function LineChartFactory(Private) {
           const height = self.baseChart.chartConfig.height;
           const circleRadius = (d.z - radii.min) / radiusStep;
 
-          return _.min([Math.sqrt((circleRadius || 2) + 2), width, height]) + (modifier || 0);
+          const base = circleRadius ? Math.sqrt(circleRadius + 2) + lineWidth : lineWidth;
```

Comment:
this is the legacy code ... no idea why its 2 ... but i can extract it to the magicNumber variable 

lineWidth should always influence the dot size ....

for example, if your line width is 10px ... you want the smaller dot to be at least 10px big ... but when your line is 1px ... your smallest dot should be 1px big as well. (so just the base value changes)